hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -80,6 +81,15 @@ public void stop() {
     if (hiveLocalDir != null) {
       hiveLocalDir.delete();
     }
+
+    // remove raw store if exists
+    try {
+      Method cleanupRawStore = HiveMetaStore.class.getDeclaredMethod("cleanupRawStore");
+      cleanupRawStore.setAccessible(true);
+      cleanupRawStore.invoke(null);
+    } catch (Exception e) {
+      // no op
+    }
```

Comment:
Could you also rewrite this one to use the `DynMethods` helpers?

```java
  private static final DynMethods.StaticMethod CLEAN_RAW_STORE = DynMethods.builder("cleanupRawStore")
      .hiddenImpl(HiveMetaStore.class)
      .orNoop()
      .buildStatic();

    CLEAN_RAW_STORE.invoke();
```

That avoids needing a try/catch block since you can replace the method with a no-op if it isn't found. And `hiddenImpl` will make it accessible.